.github/ISSUE_TEMPLATES/blog_post.md

```diff
@@ -0,0 +1,17 @@
+---
+name: Blog post proposal
+about: Use this template for proposing an idea for a new blog post.
+title: 'Blog post working title'
+labels: about-blog
```

Comment:
Small suggestion: it may be a good idea to also add another label to differentiate `blog-post-proposal`s from non blog post proposal issues/tickets. That way your project board/view won't get cluttered with other tickets and you can filter them. I'm anticipating there won't be many until we finish migrating the blog out of this repo to the future redesign repo at [sourcegraph/blog](https://github.com/sourcegraph/blog), but just wanted to throw that out there just in case. 🤓

Reply:
@bretthayes Good call! (I also ❤️ issue templates 🤓)